src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -151,19 +150,6 @@ private List<ValidationResult> validateCategoryId(final String dataFieldId,
             return Collections.singletonList(new ValidationResult(nodeType + " is not a string : " + CATEGORY_ID,
                     dataFieldId));
         }
-
-        final List<CategoryDefinition> categoryList =
-            caseDefinitionRepository.getCaseType(caseFieldDefinition.getCaseTypeId()).getCategories();
-        String categoryIdValue = categoryId.textValue();
-        final boolean caseTypeContainsKnownCategoryId = categoryList.stream()
-            .anyMatch(category ->
-                category.getCategoryId().equals(categoryIdValue));
-
-        if (!caseTypeContainsKnownCategoryId) {
-            LOG.error("{} value not recognised as a valid Case Category", CATEGORY_ID);
-            return Collections.singletonList(new ValidationResult(
-                CATEGORY_ID + " value not found", dataFieldId));
-        }
```

Comment:
Could you please provide additional details about the reason for deleting this?

Reply:
This has been branched out of CCD-4196 which has been addressed in the [PR](https://github.com/hmcts/ccd-data-store-api/pull/2232)